File: backend/src/main/java/io/github/b4official/mail/dto/response/LoginResponse.java
package io.github.b4official.mail.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.NoArgsConstructor;
import lombok.Data;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class LoginResponse {

    private String token;

    @Builder.Default
    private String type = "Bearer";


    private Long id;
    private String username;
    private String email;


}

File: frontend/src/commonMain/kotlin/io/github/b4official/mail/auth/model/LoginResponse.kt
package io.github.b4official.mail.auth.model

import kotlinx.serialization.Serializable

@Serializable
data class LoginResponse(
    val accessToken: String? = null,
    val refreshToken: String? = null,
    val tokenType: String? = null,
    val expiresIn: Long? = null,
    val userId: String? = null,
    val username: String? = null,
)
